src/tests/integration/test-data/show/ConstructorTest.test_contract_call().cse.expected

```diff
@@ -0,0 +1,309 @@
+
+┌─ 1 (root, init)
+│   k: #execute ~> CONTINUATION:K
+│   pc: 0
+│   callDepth: 0
+│   statusCode: STATUSCODE:StatusCode
+│
+│  (2233 steps)
```

Comment:
Do we know that the summaries of `set` and `add` are getting applied for sure?

Reply:
I've verified this by running with `--break-on-calls` and finding the edges where they are applied. The rules show up as `UNKNOWN`, which is something it would be nice to fix, but there are the number I would expect (5) and the calldata in the state prior to these edges (where `#execute` is at the top of the k cell) looks right.